template/src/subComponents/TextWithTooltip.native.tsx

```diff
@@ -34,21 +34,52 @@ const TextWithToolTip = (props: any) => {
                         style={style.backDrop} 
                     />
                 </TouchableWithoutFeedback>
-                <View style={[style.textContainer, {top: top,left: left}]}>
+                <View style={[style.textContainer, position]}>
                     <Text style={style.textStyle}>{props.value}</Text>
                 </View>
             </Modal>
             <TouchableOpacity ref={ref} onPress={() => {
-                    ref?.current?.measure( (fx, fy, width, height, px, py) => {
-                        // console.log('Component width is: ' + width)
+                    ref?.current?.measure( (fx, fy, localWidth, localHeight, px, py) => {
```

Comment:
remove this commented console, and give a few line explainer comment on what exactly the onPress function is doing to generate the tooltip, if there is any online explainer documentation, that's also fine.

Reply:
Removed the commented code and added some explainer. please check it, thanks